tests/browser/test/basic_playground_test.js

```diff
@@ -46,3 +51,87 @@ suite('Testing Connecting Blocks', function (done) {
     await browser.deleteSession();
   });
 });
+
+suite('Right Clicking on Blocks', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
+    browser = await testSetup(testFileLocations.playground);
+  });
+
+  test('Collapse', async function () {
+    await browser.refresh();
```

Comment:
Eric actually had thoughts on this: https://docs.google.com/document/d/1LYL6sJUQlM74XiSmAyRVadY1oMOwfqmsSpvn3CU9Khs/edit?resourcekey=0-ZU-Oy-In0vuAwFnRFwD3sQ&disco=AAAAwWOjSd8

It might be good to take a look at his options!

Reply:
After discussion: here or in a later PR, I should switch this to use `setup` and `teardown` to clear the workspace instead of reloading the entire page.